core/lib/zksync_core/src/metadata_calculator/tests.rs

```diff
@@ -166,6 +167,11 @@ async fn status_receiver_has_correct_states() {
         other_tree_health_check.check_health().await.status(),
         HealthStatus::ShutDown
     );
+
+    // Check that health checks don't prevent dropping RocksDB instances.
+    tokio::task::spawn_blocking(RocksDB::await_rocksdb_termination)
+        .await
+        .unwrap();
```

Comment:
If it ever would, would this test just hang? If so, maybe wrap it into a timeout?

Reply:
I think it could make sense to [configure test timeouts via nextest](https://nexte.st/book/slow-tests) so that we don't need to worry about each specific test.